### PR TITLE
First hello world example for Android

### PR DESCRIPTION
**Pull request by sfyz**

### CODE REVIEW

**Comment by gordienoye** [comment]
minor point about the bundle name ... should we get rid of `example` and go directly to `com.salesforce.` ?

File: Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#3F51B5</color>
```
Comment (sfyz) [inline Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml]: @sthistle Other than hardcoding the RGB values, is there a more elegant way to associate colorPrimary with the salesforce_brand_secondary, for example?

**Comment by sfyz** [comment]
@gordienoye I'm updating the package for all the classes to not have "example" in it any longer... Is that what you mean by "bundle name", or is there another place I should fix this too? Thanks for feedback!

File: Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (sthistle) [inline Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml]: I think I would add all salesforce brandable strings in this xml.
Then I would do something like:
<color name="colorPrimary">@color/salesforce_brand_primary</color>
or 
<color name="salesforce_brand_primary">@color/colorPrimary</color>
That way the developer can easily see how to theme snapins to work with their app and can easily test branding changes.

Comment (sfyz) [inline Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml]: Great. Thanks. I did it the first way, but add a  in the XML about where to find branding instructions for the SDK...

**Comment by sfyz** [comment]
OK. As far as I know, I've incorporated ALL the feedback so far. Thanks!

**Pull request merged**